on-chain/evm-contracts/test/bridge/depositERC721.js

```diff
@@ -0,0 +1,197 @@
+/**
```

Comment:
add a test to ensure extra data is retrieved as expected

Reply:
Updated issue #163 to include this